onboardProject/utils/apollo.tsx

```diff
@@ -0,0 +1,63 @@
+import { ApolloClient, HttpLink, InMemoryCache, gql } from "@apollo/client";
+import { AsyncStorage } from "react-native";
+
+export const client = new ApolloClient({
+    cache: new InMemoryCache(),
+    link: new HttpLink({
+        uri: "https://tq-template-server-sample.herokuapp.com/graphql"
+    })
+});
+
+export const storeData = async (key:string, value:string) => {
+    try{
+        await AsyncStorage.setItem(key, value);
+    }
+    catch(error){}
+};
+
+export const fetchData = async (key:string) => {
+    try{
+        const value = await AsyncStorage.getItem(key);
+        if (value !== null) {
+            return value;
+        }
```

Comment:
E se `value` for null? O que acontece?

Reply:
Hm... é, esse é o fluxo para caso consiga entrar na tela de usuários sem ter um token de autenticação. Acho que consegui modificar para ter um fluxo que leve para a tela "Não foi possível carregar o conteúdo."